in_toto/gpg/common.py

```diff
@@ -730,7 +728,13 @@ def parse_signature_packet(data, supported_signature_types=None,
     if subpacket_type == SIG_CREATION_SUBPACKET:
       info["creation_time"] = struct.unpack(">I", subpacket_data)[0]
 
-    info["subpackets"][subpacket_type] = subpacket_data
+    info["subpackets"][subpacket_type] = (subpacket_data, is_hashed)
+
+  # Warn if expiration subpacket is not hashed
+  if KEY_EXPIRATION_SUBPACKET in (info["subpackets"]):
+    if not (info["subpackets"][KEY_EXPIRATION_SUBPACKET][1]):
+      log.warning("Expiration subpacket not hashed, pgp client possibly exporting "
+        "a weakly configured key.")
```

Comment:
Please make sure to follow the [code style guidelines](https://github.com/secure-systems-lab/code-style-guidelines) (see line length max and double indentation for line continuation).